pkg/auth/providers/oidc/oidc_provider.go

```diff
@@ -317,6 +317,7 @@ func (o *OpenIDCProvider) saveOIDCConfig(config *v32.OIDCConfig) error {
 	if err := common.CreateOrUpdateSecrets(o.Secrets, convert.ToString(config.ClientSecret), field, strings.ToLower(config.Type)); err != nil {
 		return err
 	}
+	config.ClientSecret = common.GetName(config.Type, field)
```

Comment:
nit: all of these field vars pointing at different fields looks weird to me, its done that way in other areas and isn't an err, so not mandatory to change, but would be nice if it was `secretField` and `privateKeyField`

Reply:
Changed field var names to `secretField` and `privateKeyField`